index.html

```diff
@@ -1,203 +1,349 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Agasha Bornittah Portifolio</title>
-    <link rel="stylesheet" href="css/style.css">
-</head>
-<body>
-   <div class="container">
-       <header class="toolbar">
-        <a href="#" class="logo primary">Bornittah</a>
-        <nav class="menu-bars">
-         <img src="assets/icons/bars-solid.svg" class="icon" alt="icon">
-        </nav>
-    </header>
-        <section class="section section-bg-image">
-          <div class="wrapper">
-            <div class="titles">
-                <h1 class="primary-text">Hey there!, I'm Bornittah</h1>
```

Comment:
Please use the `h1` tag once on the page as it was requested by the previous reviewer. This will improve the accessibility of your page.